src/snowflake/snowpark/modin/pandas/__init__.py

```diff
@@ -165,13 +167,20 @@
 )
 
 # this must occur before overrides are applied
-_attrs_defined_on_modin_base = set(dir(modin.pandas.base.BasePandasDataset))
-_attrs_defined_on_series = set(
-    dir(Series)
-)  # TODO: SNOW-1063347 revisit when series.py is removed
-_attrs_defined_on_dataframe = set(
-    dir(DataFrame)
+_attrs_defined_on_modin_series = set()
```

Comment:
can we add some description before those code to clarify why we need those attrs related variables?

Reply:
added